docs/Local Development Install.md

```diff
@@ -0,0 +1,181 @@
+
+      docker run -d --name dcmfPostgres -p 5432:5432 -e POSTGRES_PASSWORD=dcm -e POSTGRES_USER=dcm postgres
+
+  choose your own credentials of course, please **do not** use credentails shown on the guide.
+  now connect to postgres DB either through your IDE or PGADMIN4
+
+  ![Docker desktop postgres container](images/dev/4.png "Docker postgres")
+
+  ![Postgres connection](images/dev/5.png "Docker postgres connection")
+
+- ### Running project first time for init configs
+  execute all flyaway scripts on the postgres dcm schema.
+  and create a keycloak schema (for later use by keycloak)
+  . Run the project. 
+  and
+  when you see "Completed initialization in 1ms" stop the project. 
+
+- ### Running Keycloak
+  open a new cmd file and run 
+
+      docker run -p 8888:8080 -e KEYCLOAK_ADMIN=YourAdminName -e KEYCLOAK_ADMIN_PASSWORD=YourAdminPassword quay.io/keycloak/keycloak:23.0.6 start-dev
+
+  Next go to docker and follow the link on the keycloak container. 
+
+  Login with admin credentials, under realms, click create new realm
+  import the realm-export.json on the keycloak folder.
+
+  ![Postgres connection](images/dev/5.png "Docker postgres connection")
+  
+  Modify users to you heart's content(under the users tabs, credentials for them, assing roles, etc). 
+
+  **Remember you need to have a user role on all users, it can be ADMIN, CUSTOMER, SUPPLIER**
+  failing to have one of these roles won't let the user login in the app.
+
+  [Download keycloak realm json](realm-export.json)
+
+- ### Fetching the keycloak client credential
+  before booting the project again navigate to dcmauth client on keycloak panel and copy the client secret under credentials.
+  open your application.yaml and place it on the dcmsecr section.
+  
+  after that run the project and in postman you should be able to login on the token endpoint with the credentials you modified on keycloak!
+
+  ![Postman](images/dev/6.png "Postman login")
+
+- ### Configure postman collection requests!
+  for the other requests on postman you need to alter the authorization tab.
+  check the config on the images provided.
+
+  the access token url should be either the backend or in last resort if it fails, this one 
+  **http://localhost:8888/auth/realms/dcm/protocol/openid-connect/token** 
+  if an authorization error occurs, it most likely is the url error of keycloak, newer versions of keycloak you need to remove **/auth/** from the authorization url.
+
+  ![Postman](images/dev/7.png "Postman config")
+
+  ![Postman](images/dev/8.png "Postman config")
+
+  ![Postman](images/dev/9.png "Postman config")
+
+  ![Postman](images/dev/10.png "Postman config")
+
+ 
+
+- ### Run the front-end
+  when postman is working, you need to open the front end on your IDE of choice and run on a terminal inside the front-end folder, make sure you have NodeJS installed on your machine.
+
+      yarn install --force --legacy-peer-deps
+
+      yarn 
+    
+    or
```

Comment:
I think you switched to `npm` only right?

Reply:
@SebastianBezold 
Is the packet manager relevant for eclipse or or there any restrictions in this regard?

@Ruskyy 
Please address Sebastian's concern.